contracts/epochs/FlowEpoch.cdc

```diff
@@ -94,13 +106,13 @@ pub contract FlowEpochs {
         // The resulting public keys from the DKG process, encoded as by the flow-go
         // crypto library, then hex-encoded.
         // TODO: define ordering
-        // TODO: which is group public key
+        // Group public key is the last element
         dkgPubKeys: [String],
 
         // The result of the QC aggregation process. Each element contains all the votes
         // received for a particular cluster, comma-separated.
         // TODO: define ordering
-        clusterQCs: [String]
+        clusterQCs: [FlowEpochClusterQC.ClusterQC]
```

Comment:
adjust the comment here, what is the difference between collectorsClusters  and clusterQCs